resources/git-workflow.md

```diff
@@ -1,11 +1,54 @@
 # 🛶 Git Workflow
 
 Teams at Launch Pad will use the git version control system and GitHub website
-to build software collaboratively. This document describes a recommended GitHub
-workflow for Launch Pad teams, as well as a basic intro on how to execute that
-workflow yourself.
+to build software collaboratively. This document starts with a "zero-to-hero"
+guide to Git and concludes with a recommended GitHub workflow for Launch Pad
+teams as well as a basic intro on how to execute that workflow yourself.
 
-## Updating Your Copy of the Codebase
+If you're familiar with Git, feel free to jump ahead to the [recommended workflow](#recommended-workflow) section.
+
+## Git Zero to Hero
+
+### What is Git?
+
+Git is fundamentally a *version control system* - a tool that lets you maintain
+a timeline of changes that have occured in your project. Having a version
+history is great because you get to freely make changes without ever worrying
+about losing prior work.
+
+::: tip
+Git is not the same thing as Git*Hub*. GitHub is a platform that lets you store
+(and do other things) your Git-tracked folders (known as repositories) on a
+remote server. This lets you easily share these repositories across team
+members.
+:::
+
+On top of keeping history, Git comes with a set of features that make it truly
+great for collaborative work. Git allows you to create *branches*, alternative
+versions of your code, that could later be *merged* back to the main history.
+We'll get to this in a bit - let's first talk about how to get your repository
+set up.
+
+### Creating a Repository
+
+TODO: Should this be more conceptual or more tutorial-like?
```

Comment:
I would replace this with "What is a repository", since very few people will create a repository